runtime/tests/jvmtitests/agent/tests.c

```diff
@@ -95,6 +95,7 @@ static jvmtiTest jvmtiTestList[] =
 	{ "rc019a", rc019a, "com.ibm.jvmti.tests.redefineClasses.rc019a", "RedefineClasses" },
 	{ "rc019b", rc019b, "com.ibm.jvmti.tests.redefineClasses.rc019b", "RedefineClasses" },
 	{ "rc020", rc020, "com.ibm.jvmti.tests.redefineClasses.rc020", "RedefineClasses" },
+	{ "rc021", rc021, "com.ibm.jvmti.tests.redefineClasses.rc021", "RedefineClasses" },
```

Comment:
```suggestion
	{ "rc021", rc021, "com.ibm.jvmti.tests.redefineClasses.rc021", "RetransformClasses" },
```

Reply:
Renamed test to rtc003 - the distinction is strictly notational. Retransform is just a much easier way to do a NOP HCR than trying to find the class bytes.